### DISCUSSION TITLE
Performance

### DISCUSSION THREAD

**Discussion by Midasboi132** [rafaelvcaetano/melonDS-android#1339] [Q&A]
I have a beefy phone. I expected the emulator to run about as fast as drastic. But it seems to run like it's actual hardware. Is there anything I can do to increase the fps in games like call of duty?

**Comment by Unknowncat1991** [comment]
![IMG_20250304_023125](https://github.com/user-attachments/assets/cf645d6c-a1d5-438d-810a-8b0295b09fd1)
melonDS might improve performance if ever the developer might do that just like DraStic did idk but maybe 😅

**Reply by lifewlevi** [reply]
i just wish someone would revive drastic or something at this point, every single game works better on Drastic, High Resolution 3D Rendering doesn't even decrease performance, it's just perfect meanwhile if you increase resolution on melonDS...
i hope the guy(s) behind melonDS work it out and do it the way Drastic dev(s) did.
